RNVectorIcons.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.platform       = :ios, "7.0"
   s.source         = { :git => "https://github.com/oblador/react-native-vector-icons.git", :tag => "v#{s.version}" }
   s.source_files   = 'RNVectorIconsManager/**/*.{h,m}'
-  s.resources      = "Fonts/*.ttf"
+  s.resources      = "Fonts/*.(o|t)tf"
```

Comment:
Is this change needed? Seems like FA5 uses `.ttf` too?

Reply:
This is true, my first design was using the desktop version of the font (which is .otf) but I later moved to the npm hosted version and just forgot about this but I can change it.